Sources/Plasma/FeatureLib/pfPatcher/pfPatcher.cpp

```diff
@@ -264,6 +264,13 @@ class pfPatcherStream : public plZlibStream
         return retVal;
     }
 
+    void Close()
```

Comment:
is this `override`?

Reply:
There is no more `virtual Close();` as of #1448.

Reply:
Ah okay